test/integration/lib/test-actions/index.ts

```diff
@@ -119,11 +119,8 @@ async function _startAndAcceptInteraction(
   assert(interactId)
 
   // Accept
-  const url = new URL(senderWalletAddress.authServer)
-  url.port = '3109'
-
   const acceptResponse = await fetch(
-    `${url.toString()}grant/${interactId}/${nonce}/accept`,
+    `${deps.sendingASE.config.interactionServer}/grant/${interactId}/${nonce}/accept`,
```

Comment:
final thing, I think we just need to expose `interactionServer` on the `TestConfig` interface